org.mwc.debrief.montemedia/src/MovieMaker.java

```diff
@@ -0,0 +1,30 @@
+
+import org.monte.moviemaker.MovieMakerMain;
+
+/*
+ * @(#)MovieMaker.java  1.0  2011-03-19
+ * 
+ * Copyright (c) 2011 Werner Randelshofer, Immensee, Switzerland.
+ * All rights reserved.
+ * 
+ * You may not use, copy or modify this file, except in compliance with the
+ * license agreement you entered into with Werner Randelshofer.
+ * For details see accompanying license terms.
+ */
+
+/**
+ * {@code MovieMaker}.
+ *
+ * @author Werner Randelshofer
+ * @version 1.0 2011-03-19 Created.
+ */
+public class MovieMaker {
```

Comment:
Hello @saulhidalgoaular  - is there a use-case when we run this app?

Reply:
This is a tool to test manually the class that creates .mov files. You can upload some images and an audio and it creates a mov file from that. We don't have an use case in Debrief for this functionality. I am removing it.

By the way, there are many other classes like this one. Would you like me to delete them too? They are some tools to test partial functionality of the library.